Automattic/pocket-casts-ios
# End of Year: Implements design feedback for the listened numbers story

| 📘 Project: #374 | 🛫 Depends on: #561 |
|:---:|:---:|

This addresses the following items:
- Moves "kept" to a new line
- Updates the fonts
- Adjusts spacing between the covers
- I did not scale the covers up in size because the image assets we have became blurry.

## Screenshots

### Design vs Implementation Comparison

<p align="center"><img src="https://user-images.githubusercontent.com/793774/205229629-8e5b7841-b6c8-48a5-876e-0d07dd4687fc.png" width="100%" /></p>

### Devices
| iPhone SE - 14.1 | iPhone 13 | iPhone 14 Pro Max |
|:---:|:---:|:---:|
|<img src="https://user-images.githubusercontent.com/793774/205229714-bb38735c-70f1-450c-a049-3ab3d7b675e8.png" width="320" />|<img src="https://user-images.githubusercontent.com/793774/205229715-0113c896-303e-4fcd-9651-070c92dbe439.png" width="320" />|<img src="https://user-images.githubusercontent.com/793774/205229711-c248d155-c508-4877-928b-c418fa5c20d6.png" width="320" />|

## To test

1. Launch the app
2. Tap on Profile > End of Year card
3. Tap to the 5th screen
4. ✅ Verify it looks good and matches the designs
5. ✅ Repeat the steps of various screen sizes

## Checklist
